lametro/templates/lametro/legislation.html

```diff
@@ -108,7 +108,12 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
                                         {% endif %}
                                     </td>
                                     <td class="small">
-                                        {{action.organization.link_html|safe}}
+                                      {% if action|get_event_occured|get_event_status == 'cancelled' %}<strike>{% endif %}
```

Comment:
calling this twice will lead to duplicate queries. how about using some `with` logic to grab the event? here's an example from the index page:

https://github.com/datamade/la-metro-councilmatic/blob/25ab4a6489e0862b06d714c8a7dc529130dcb08f/lametro/templates/lametro/index.html#L72-L74

(the partial can then reference the meeting object like `{{ meeting }}`.)